s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
I feel a bit like an intermediary here :)

There are three important things to track, the upstream version if building from a tarball, package, or release tag, the upstream commit if not, and the cgl-docker-lib commit (which puts the image in quay.io via the Jenkins build).  I'm not sure which should go in the name and which in the tag.

I would propose, for building from a tarball, package, or release tag
```
source directory = ${upstream name}
name = quay.io/ucsc_cgl/${upstream name}
tag = ${upstream version}--${cgl-docker-lib commit}
```

and for building from upstream source
```
source directory = ${upstream name}-dev
name = quay.io/ucsc_cgl/${upstream name}-dev
tag = ${upstream commit}--${cgl-docker-lib commit}
```